doc/rnc/lifecycle_configuration.rnc

```diff
@@ -24,11 +24,21 @@ Expiration =
   element ExpiredObjectDeleteMarker { xsd:boolean }
 
 Filter =
-  element Prefix { xsd:string }? &
-  element Tag {
-    element Key { xsd:string } &
-    element Value { xsd:string }
-  }*
+  (
+    element Prefix { xsd:string }? &
+    element Tag {
+      element Key { xsd:string } &
+      element Value { xsd:string }
+    }
+  ) | (
+    element And {
+      element Prefix { xsd:string }? &
+      element Tag {
+        element Key { xsd:string } &
+        element Value { xsd:string }
+      }*
```

Comment:
In an `And`, there must be at least 2 filters. Even if there is a `Prefix`, there must be at least one `Tag`. So we should use `+` (instead of `*`).

Reply:
Done.